### DISCUSSION TITLE
Here's a fix for sslmode on Postgres Upsert/Load from Existing Vector Component

### DISCUSSION THREAD

**Discussion by arberrexhepi** [FlowiseAI/Flowise#1133] [General]
file> packages/components/nodes/vectorstores/Postgres_Upsert/Postgres_Upsert.ts

because most third party deploy services seem to require sslmode=require, it does not connect without this fix. Please review and implement!
     
   const postgresConnectionOptions = {
            ...additionalConfiguration,
            type: 'postgres',
            host: nodeData.inputs?.host as string,
            port: nodeData.inputs?.port as number,
            username: user,
            password: password,
            database: nodeData.inputs?.database as string,
            ssl: true  // <-- Added this line
        }

  const poolOptions = {
            host: postgresConnectionOptions.host,
            port: postgresConnectionOptions.port,
            user: postgresConnectionOptions.username,
            password: postgresConnectionOptions.password,
            database: postgresConnectionOptions.database,
            ssl: true  // <-- Added this line
        }

**Comment by HenryHengZJ** [comment]
there was a PR that does this - https://github.com/FlowiseAI/Flowise/pull/1107 , have not pushed to new release yet

**Reply by arberrexhepi** [reply]
great, looking forward to it!
